Certification test for Postgres config store

shivamkm07 opened the pull request:
# Description

This PR adds certification test for postgres config store.

## Issue reference

We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: #2743 

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [ ] Code compiles correctly
* [ ] Created/updated tests
* [ ] Extended the documentation / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_

shivamkm07 commented:
This PR should be merged only after the conformance PR #2740 is merged, as it is rebased on the top of it. But it is open for review.

berndverst commented on tests/certification/configuration/postgres/postgres_test.go:
```diff
@@ -0,0 +1,314 @@
+	currentGrpcPort := ports[0]
+	currentHTTPPort := ports[1]
+
+	watcher1 := watcher.NewUnordered()
```
Configuration stores don't guarantee order of updates - so this is generally ok. Though since we are only dealing with updates to the same key where order does actually matter.. this probably needs to be updated to `Ordered()` for the sake of this test.

shivamkm07 replied:
Yes config stores don't guarantee order of updates. For a single key, there would be just one update notification and so ordered and unordered are same. 
I have updated the test to subscribe to multiple keys for more robust test.

shivamkm07 commented on tests/certification/configuration/postgres/postgres_test.go:
```diff
@@ -0,0 +1,314 @@
+					Items: castConfigurationItems(items),
+				}
+				updateEventInJson, err := json.Marshal(updateEvent)
+				assert.NoError(t, err)
```
Done

shivamkm07 commented:
@berndverst @ItalyPaleAle I have rebased the PR to master and since now #2740 is merged, this PR is open to review(and if possible merge). 
I have also added more scenarios to test get with multiple versions.
